db/fixtures/ae_datastore/ManageIQ/ConfigurationManagement/AnsibleTower/Service/Provisioning/StateMachines/Provision.class/__methods__/update_serviceprovision_status.rb

```diff
@@ -13,7 +13,12 @@
 # Get status from input field status
 status = $evm.inputs['status']
 
-# Update Status for on_entry,on_exit
-if $evm.root['ae_result'] == 'ok' || $evm.root['ae_result'] == 'error'
-  prov.message = status
-end
+# Update Status Message
+updated_message  = "Server [#{$evm.root['miq_server'].name}] "
+updated_message += "Service [#{prov.destination.name}] "
+updated_message += "Step [#{$evm.root['ae_state']}] "
+updated_message += "Status [#{status}] "
+updated_message += "Message [#{prov.message}] "
+updated_message += "Current Retry Number [#{$evm.root['ae_state_retries']}]" if $evm.root['ae_result'] == 'retry'
+prov.miq_request.user_message = updated_message
+prov.message = status
```

Comment:
@mkanoor We're intentionally setting the message each time update_provision_status is called. The retry information is important for the user.